pkg/models/v1/models.go

```diff
@@ -504,6 +504,10 @@ func UpdateCertificateRequestStatus(request *CertificateRequest, state Certifica
 	return nil
 }
 
+func GetCertID(identityCert *pb.Certificate) string {
```

Comment:
I think we should add a comment above this method to make it clear what it's doing, since it's not obvious from the name that it's returning the capital hex encoded string version of the serial number, and also why it's necessary (e.g. the certificate IDs are the string encoded version of the certificate serial number which match what is displayed by sectigo).